.gitignore

```diff
@@ -3,4 +3,12 @@
 
 node_modules/
 examples/
-npm-debug.log
\ No newline at end of file
+npm-debug.log
+
+dist/
```

Comment:
I don't think Oli usually excludes dist, because it's used to download a specific version for GitHub

Reply:
Oh my bad. It was supposed to be for my local text editor.